pkg/app/api/service/webservice/service.proto

```diff
@@ -401,3 +405,15 @@ message ListAPIKeysRequest {
 message ListAPIKeysResponse {
     repeated model.APIKey keys = 1;
 }
+
+message GetInsightDataRequest {
+    string application_id = 1 [(validate.rules).string.min_len = 1];
+    pipe.model.InsightMetricsKind metrics_kind = 2;
+    pipe.model.InsightStep insight_step = 3;
+    int64 first_date = 4;
+    int64 last_date = 5;
+}
+message GetInsightDataResponse {
+    int64 updated_at = 1;
+    repeated pipe.model.InsightDataPoint data_points = 2;
+}
```

Comment:
Please add an empty line at the end of the file.

Reply:
Do we have lint for this kind of problem? Missing lines at the end of the file are a regular issue. 👀

Reply:
We were talking about that adding a Kapetanios plugin to perform the linter for protobuf before, but still not addressed it...

Reply:
got it 🙆‍♂️